### PR TITLE
Update error status with error message

### PR DESCRIPTION
**Pull request by swaroop-osec**
- Minor fix to display error response instead of just status code.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -878,10 +878,13 @@ pub async fn send_job_to_remote(
     if response.status().is_success() {
         println!("Successfully sent job to remote");
         Ok(())
+    }  else if response.status() == 409 {
+        println!("We have already processed this request");
```
Comment (jarry-xiao) [inline src/main.rs:882]: Sorry to nit, but I think a better message here would be something like:
```
Program $PROGRAM_ID has already been verified ✅
Program hash: $HASH
```

The main idea being that if someone attempts to run this, the output is extremely clear

Comment (jarry-xiao) [reply src/main.rs:882]: Another thing, you probably want to confirm that the network is mainnet before they enter this function

Comment (swaroop-osec) [reply src/main.rs:882]: We just drop duplicate requests even if the program isn't verified so it's not possible to say that the program is verified. And for 2 I'll add a check before entering the function.

Comment (jarry-xiao) [reply src/main.rs:882]: can't you just hit the status endpoint?

Comment (swaroop-osec) [reply src/main.rs:885]: OK Now I've update like this 

Duplicate request & verified

```
Program $PROGRAM_ID has already been verified. ✅
On Chain Hash: $ON_CHAIN_HASH
Executable Hash: $BUILD_HASH
```
Duplicate request & not verified

```
We have already processed this request.
```


Wdyt about this?

**Pull request closed without merging**